pybliometrics/scopus/serial_title.py

```diff
@@ -264,3 +274,54 @@ def _parse_list(d, metric):
         return sorted(set(values))
     except (KeyError, TypeError):
         return None
+
+
+def _get_all_cite_score_years(self, named_info_list, named_rank_list, data) -> Optional[List[NamedTuple]]:
+    """Auxiliary function to get all information contained in cite score information list for the `CITESCORE` view"""
+    data = data.get('citeScoreYearInfo', [])
+
+    new_data = []
+    # Iterate through years
+    for d in data:
+        citeScoreInfo = d.get('citeScoreInformationList', [])[
+            0].get('citeScoreInfo', [])[0]
+        # Iterate through subject ranks
+        subject_rank = [named_rank_list(subjectcode=int(subject['subjectCode']),
+                        rank=int(subject['rank']),
+                        percentile=int(subject['percentile']))
+                        for subject in citeScoreInfo['citeScoreSubjectRank']]
+        # Create named tuple with info
+        Citescoreinfolist_year = named_info_list(year=int(d['@year']),
+                                                 status=d['@status'],
+                                                 documentcount=int(
+                                                     citeScoreInfo['scholarlyOutput']),
+                                                 citationcount=int(
+                                                     citeScoreInfo['citationCount']),
+                                                 citescore=make_float_if_possible(
+                                                     citeScoreInfo['citeScore']),
+                                                 percentcited=int(
+                                                     citeScoreInfo['percentCited']),
+                                                 rank=subject_rank)
```

Comment:
Please use only one level of indentation more than line 294, and then one line per key